Documentation/Patterns.md

```diff
@@ -192,7 +192,7 @@ struct Empty { };
 static_assert(AssertSize<Empty, 1>());
 ```
 
-## String View Literals
+## DeprecatedString View Literals
```

Comment:
:thinking:

Reply:
Thanks, undid all changes in `Documentation/`.